services/influxdb/service_test.go

```diff
@@ -1263,19 +1260,42 @@ func (c influxDBClient) Ping(ctx context.Context) (time.Duration, string, error)
 	}
 	return 0, "testversion", nil
 }
+
 func (c influxDBClient) Write(bp influxcli.BatchPoints) error {
 	if c.WriteFunc != nil {
 		return c.WriteFunc(bp)
 	}
 	return nil
 }
+
+func (c influxDBClient) WriteV2(w influxcli.FluxWrite) error {
+	if c.WriteV2Func != nil {
+		return c.WriteV2Func(c.clusterName, w)
+	}
+	return nil
+}
+
 func (c influxDBClient) Query(q influxcli.Query) (*influxcli.Response, error) {
 	if c.QueryFunc != nil {
 		return c.QueryFunc(c.clusterName, q)
 	}
 	return &influxcli.Response{}, nil
 }
 
+func (c influxDBClient) QueryFlux(q influxcli.FluxQuery) (flux.ResultIterator, error) {
+	if c.FluxQueryFunc != nil {
+		return c.FluxQueryFunc(c.clusterName, q)
+	}
+	return nil, nil
+}
+
+func (c influxDBClient) QueryFluxResponse(q influxcli.FluxQuery) (*influxcli.Response, error) {
```

Comment:
I don't see a real implementation of this?

Reply:
It isn't part of this PR it was already in master.
https://github.com/influxdata/kapacitor/blob/226f1cad9207676c5fd86699b27620c31fbff6ae/influxdb/client.go#L616